internal/provider/datasources/data_source_api_tokens_test.go

```diff
@@ -0,0 +1,175 @@
+package datasources_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients/iam"
+
+	astronomerprovider "github.com/astronomer/terraform-provider-astro/internal/provider"
+	"github.com/astronomer/terraform-provider-astro/internal/utils"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAcc_DataSourceApiTokens(t *testing.T) {
+	tfVarName := "test_data_api_tokens"
+	tfOrganizationId := os.Getenv("HOSTED_ORGANIZATION_ID")
+	tfWorkspaceId := os.Getenv("HOSTED_WORKSPACE_ID")
+	tfDeploymentId := os.Getenv("HOSTED_DEPLOYMENT_ID")
+	tfOrgOnly := true
```

Comment:
nit: not needed

Reply:
why is this not needed? i used all of these ids as a check in `checkApiTokens` function